### PR TITLE
lodash: allow get to receive $ReadOnlyArray

### PR DESCRIPTION
**Pull request by lukeapage**

### CODE REVIEW

File: definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js [hunk 1]
```diff
@@ -1039,8 +1039,8 @@ declare module "lodash" {
     at(object?: ?Object, paths: Array<string>): Array<any>;
     create<T>(prototype: T, properties: Object): $Supertype<T>;
     create(prototype: any, properties: void | null): {};
-    defaults(object?: ?Object, ...sources?: Array<Object>): Object;
-    defaultsDeep(object?: ?Object, ...sources?: Array<Object>): Object;
+    defaults(object?: ?Object, ...sources?: Array<?Object>): Object;
```
Comment (GAntoine) [inline definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js]: I

File: definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js [hunk 2]
```diff
@@ -1039,8 +1039,8 @@ declare module "lodash" {
...
+    defaultsDeep(object?: ?Object, ...sources?: Array<?Object>): Object;
```
Comment (GAntoine) [inline definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js]: I think you made these changes in another PR which I just merged in. You'll need to remove them from this one before I can accept it!

**Comment by lukeapage** [comment]
@GAntoine fixed

**Comment by jedwards1211** [comment]
Does no one realize that lodash accepts numbers in path arrays?

**Comment by lukeapage** [comment]
@jedwards1211 https://github.com/flow-typed/flow-typed/pull/2444

**Comment by jedwards1211** [comment]
@lukeapage oh, thanks!

**Pull request merged**